src/shared/user-record.h

```diff
@@ -41,6 +36,13 @@ static inline bool gid_is_container(gid_t gid) {
         return uid_is_container((uid_t) gid);
 }
 
+typedef struct login_defs {
+        uid_t system_uid_max;
+        gid_t system_gid_max;
+} login_defs;
```

Comment:
btw, i think we kinda followed the rules that typedefs either use CamelCase naming, or carry the "sd_" prefix. but not sure, maybe we don't follow that rule strictly, but i think this would be an outlier here...